core/storage/topic/gae_models.py

```diff
@@ -422,9 +425,46 @@ def get_export_policy(cls) -> Dict[str, base_models.EXPORT_POLICY]:
             'thumbnail_filename': base_models.EXPORT_POLICY.NOT_APPLICABLE,
             'thumbnail_bg_color': base_models.EXPORT_POLICY.NOT_APPLICABLE,
             'version': base_models.EXPORT_POLICY.NOT_APPLICABLE,
-            'url_fragment': base_models.EXPORT_POLICY.NOT_APPLICABLE
+            'url_fragment': base_models.EXPORT_POLICY.NOT_APPLICABLE,
+            'story_exploration_mapping':
+                base_models.EXPORT_POLICY.NOT_APPLICABLE
         })
 
+    @classmethod
+    def get_story_exploration_mappings(cls) -> List[Dict[str, List[str]]]:
```

Comment:
get_all_story_exploration_mappings()

Reply:
This is no longer relevant since the projection query idea for the new field was dropped (as `JsonProperty` couldn't be indexed).